Calculating.java

```diff
@@ -0,0 +1,13 @@
+package UnitOmega13;
+
+public class Calculating {
+
+	public long exponentiation(int number, int degree){
+		int result = 1;
+		for (int originalnumber = 0; originalnumber < degree; originalnumber++) {
```

Comment:
use camelCase with name of the parameters
https://mate-academy.github.io/style-guides/java/java.html#s5.1-identifier-names